tests/ext/reset_configured_overrides.phpt

```diff
@@ -0,0 +1,55 @@
+--TEST--
```

Comment:
Do you think that makes sense to add a test to make sure that the exception is not thrown? Something like

```
--TEST--
Check tracing of non existing classes, methods and functions.
--FILE--
<?php

class ClassWithNoMethods
{
}

dd_trace("ClassNotExist", "method" ,function(){});
dd_trace("ClassWithNoMethods", "method", function(){});
dd_trace("function_not_exist", function(){});

?>
--EXPECT--

```

Reply:
Ops I missed two files when submitting this PR. I'll do better next time. One of those files checked exactly what, you've been proposing.